website/docs/docs/deploy/job-notifications.md

```diff
@@ -1,33 +1,83 @@
+:::
+
+### Prerequisites 
+- You must be an administrator of the Slack workspace. 
+- You must be an account admin to configure Slack notifications in dbt Cloud. For more details, refer to [Users and licenses](/docs/cloud/manage-access/seats-and-users).
+
+### Set up the Slack integration
+
+1. Go to your [User profile](https://cloud.getdbt.com/settings/profile) page and scroll to the **Linked accounts** section.
+1. In the **Linked accounts** section, find the Slack application and click **Link**.
+   <Lightbox src="/img/docs/dbt-cloud/Link-your-Slack-Profile.png" width="75%" title="Link for the Slack app"/>
+1. Allow dbt Labs to access the Slack workspace. If you are a member of multiple workspaces, you can select the appropriate workspace from the dropdown menu in the upper right corner.
+   <Lightbox src="/img/docs/dbt-cloud/Allow-dbt-to-access-slack.png" width="75%" title="Allow dbt access to Slack"/>
+
+### Configure Slack notifications
+
+1. From the gear menu in the upper right corner, choose **Notification settings**. 
+1. Select **Slack notifications** in the left sidebar. 
+1. Select the **Notification channel** (in the dropdown) you want to send the job run notifications to. 
+    <Lightbox src="/img/docs/deploy/example-notification-slack-channels.png" width="75%" title="Example of the Notification channel dropdown"/>
+1. Select the **Environment** (in the dropdown) for the jobs you want to receive notifications about. 
+1. Click **Edit** to configure the Slack notification settings. Choose one or more of the run statuses (**Succeeds**, **Fails**, **Is Canceled**) for each job you want to receive notifications about.
+1. When you're done with the settings, click **Save**.
+    
+    To send alerts to another Slack channel, select another **Notification channel** from the dropdown, **Edit** those job notification settings, and **Save** the changes.
+
+    To set up alerts on jobs from a different environment, select another **Environment** from the dropdown, **Edit** those job notification settings, and **Save** the changes.
+
+    <Lightbox src="/img/docs/deploy/example-slack-notification-settings-page.png" width="100%" title="Example of the Slack notifications page"/>
+
+### Disable the Slack integration
+
+1. Go to your [User profile](https://cloud.getdbt.com/settings/profile) page and scroll to the **Linked accounts** section.
+1. Find the Slack application in the **Linked accounts** section, click the trash can icon, and click **Unlink**. You will no longer receive Slack notifications. However, other account admins might still receive the notifications. If they want to stop receiving the notifications, they can update their [Slack notification configuration](#configure-slack-notifications) or disable the Slack integration on their dbt Cloud account. 
```

Comment:
For the second bullet, how about something like the following? I think it's a bit more accurate:

> Find the Slack application in the **Linked accounts** section, click the trash can icon, and click **Unlink**. Channels **you configured** will no longer receive Slack notifications. Channels configured by other users will continue to receive Slack notifications as long as those users still have active Slack integrations. To migrate ownership of a Slack channel notification configuration, have another account admin edit the configuration.

Reply:
thank you!